docs: add meeting minutes for 2019-10-22 (#8)
hamburg-js/minutes: 2019-10-22.md

# Meta-Meetup 2019-10-22

## Agenda

- Reflection of the last meetup
- Action items from last meeting:
  - create print templates for signage (Angela)
  - Ask Feli for contacts to dev advocates (Lea)
- Meetup.com pricing model changes
- Sharing Twitter credentials and strategy
- Create list of next meetup dates
- Plan the next meetup

## Participants

...

## Minutes

> ### Reflection of the last meetup (Finanzcheck)

Reviewer: The last meetup has been at Wunder.

# Meta-Meetup 2019-10-22

## Agenda

- Reflection of the last meetup
- Action items from last meeting:
  - create print templates for signage (Angela)
  - Ask Feli for contacts to dev advocates (Lea)
- Meetup.com pricing model changes
- Sharing Twitter credentials and strategy
- Create list of next meetup dates
- XMAS special meetup
- Plan the next meetup

## Participants

Björn Brauer, Daniel Dembach, Angela Galliat, Emanuel Kluge, Deniz Okcu, Lea Rosema

## Minutes

### Reflection of the last meetup (Wunder)

- __(i)__ We had only one proposal available for that day, thankfully Martin spontaneously jumped in to give a second talk
- __(d)__ We will try to increase our communication efforts through twitter & slack and post more regularly call for papers
- __(i)__ The talks and location (Wunder) were great and feedback was positive
- __(i)__ Sometimes Q&A sessions drag out between a few participants and the speaker by sliding into a back and forth questioning

> - __(d)__ We will try to be more aware of than, when that happens and redirect it to the break after the Q&A has ended

Author: ```suggestion
- __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversions during the breaks
```

> - __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversions during the breaks

Reviewer: ```suggestion
- __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversations during the breaks
```
